bazel/devbuild/prepare_developer_workspace.sh.in

```diff
@@ -18,14 +18,17 @@ bootstrapper=$(realpath @@BOOTSTRAPPER@@)
 stat "${bootstrapper}" >> /dev/null
 upgrade_agent=$(realpath @@UPGRADE_AGENT@@)
 stat "${upgrade_agent}" >> /dev/null
-cli_enterprise=$(realpath @@CLI_ENTERPRISE@@)
-stat "${cli_enterprise}" >> /dev/null
-cli_oss=$(realpath @@CLI_OSS@@)
-stat "${cli_oss}" >> /dev/null
+cli=$(realpath @@CLI@@)
+stat "${cli}" >> /dev/null
 cdbg=$(realpath @@CDBG@@)
 stat "${cdbg}" >> /dev/null
 container_sums=$(realpath @@CONTAINER_SUMS@@)
 stat "${container_sums}" >> /dev/null
+edition=$(cat @@EDITION@@)
+if [[ ${edition} != "enterprise" && ${edition} != "oss" ]]; then
+  echo "Error: invalid edition: ${edition}"
+  exit 1
+fi
```

Comment:
What's the idea using `@@EDITION@@` here? The check is never false when called from `//:devbuild` since bazel already checks for correct values:
```
Error setting @//bazel/settings:cli_edition: invalid value 'enterpris'. Allowed values are ["", "enterprise", "oss"]
```
Is the idea just to have a reminder in the form of the `echo` below?

Reply:
Yes. I want to be able to know what edition was chosen in the alias. I can remove the check and simply echo whatever edition was chosen down below. How does that sound?

Reply:
I think it's good to know the single place where the validation happens if it needs to be adjusted, therefore this sounds good to me :+1: